Backpack-SwiftUI/Calendar/Classes/CalendarHeader.swift

```diff
@@ -34,7 +34,7 @@ struct CalendarHeader: View {
             }
             Divider()
         }
-        .accessibilityHidden(true)
+        .accessibilityChildren { EmptyView().accessibilityHidden(true) }
```

Comment:
```suggestion
         .accessibilityElement(children: .ignore)
```
Wouldn't this have the same effect? To avoid using EmptyView

Reply:
I tried this and even that this header is not readable anymore, this solution is not ideal as it left the whole empty block and it could be confusing

https://github.com/user-attachments/assets/3b32d140-6bf3-48db-9b9d-41025b523720